sql/parser/sql.y

```diff
@@ -54,8 +55,8 @@ package parser
   limit          *Limit
   targetList     TargetList
   targetListPtr  *TargetList
-  privilegeType  PrivilegeType
-  privilegeList  PrivilegeList
+  privilegeType  structured.Privilege
```

Comment:
that's fair. Another advantage of ripping them out of proto is that I can make the constants uint32, removing a lot of annoying casting in privilege.go. I'll put a commit together on this PR